csi/hack/install_modelregistry.sh

```diff
@@ -0,0 +1,48 @@
+set -e
```

Comment:
yeah makes sense, I used `hack` just because I saw that from kserve https://github.com/kserve/kserve/tree/master/hack :laughing: 

Updated `hack` --> `scripts` :rocket: